include/dlpack/dlpack.h

```diff
@@ -81,6 +81,8 @@ typedef enum {
   kDLUInt = 1U,
   kDLFloat = 2U,
   kDLBfloat = 4U,
+  kDLComplex = 8U,    // C/C++/Python layout (compact struct per complex number)
```

Comment:
do do not need to assign bitmasking. the number 3U was reserved for opaque handle, we can directly assign 5 here

Reply:
OK, changed in b6a64fa. @tqchen What's it for? I don't find it documented anywhere.